api/Node.json

```diff
@@ -171,7 +171,7 @@
               "version_added": null
             },
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
This `baseURIObject` entry should definitely be removed. Want to do that in a separate PR?